test/fixtures/skip/default/expected.js

```diff
@@ -0,0 +1,6 @@
+var _div$statics = ["__skip", "true"];
```

Comment:
How do we feel about these being included in the attributes the node is rendered with? Should they be excluded?